packages/browser-crawler/src/internals/browser-launcher.ts

```diff
@@ -1,29 +1,27 @@
 import os from 'os';
 import ow from 'ow';
 import { ENV_VARS } from '@apify/consts';
-import { BrowserPluginOptions } from 'browser-pool';
-import { isAtHome } from '../utils';
-import { BrowserPlugin } from './browser_plugin';
-import { Constructor } from '../typedefs';
-import { StealthOptions } from '../stealth/stealth';
+import {
+    Dictionary,
+    Constructor,
+    isAtHome,
+} from '@crawlers/core';
+import {
+    BrowserPlugin,
+    BrowserPluginOptions,
+} from 'browser-pool';
 
 export interface BrowserLaunchContext<TOptions> extends BrowserPluginOptions<TOptions> {
-
     /**
      * If `true` and `executablePath` is not set,
-     * Playwright will launch full Google Chrome browser available on the machine
+     * the launcher will launch full Google Chrome browser available on the machine
      * rather than the bundled Chromium. The path to Chrome executable
      * is taken from the `APIFY_CHROME_EXECUTABLE_PATH` environment variable if provided,
      * or defaults to the typical Google Chrome executable location specific for the operating system.
      * By default, this option is `false`.
      * @default false
      */
     useChrome?: boolean;
-
-    /**
-     * By default this function uses `require("playwright").chromium`.
-     * If you want to use a different browser you can pass it by this property as `require("playwright").firefox`
-     */
```

Comment:
same here, we should adjust the comment if its dated, not remove it

Reply:
It doesn't really make sense to have these on the base interface... Puppeteer doesn't have a browserType iirc, playwright does

Reply:
Oh, I see, for puppeteer we have it typed as unknown due to puppeteer-extra and such. Thoughts?

Reply:
> It doesn't really make sense to have these on the base interface... Puppeteer doesn't have a browserType iirc, playwright does

Then let's move it to the place where it makes sense